lms/static/js/student_profile/views/learner_profile_factory.js

```diff
@@ -42,6 +42,16 @@
                 accountSettingsPageUrl: options['account_settings_page_url']
             });
 
+            var profileImageFieldView = new FieldsView.ImageFieldView({
+                    model: accountSettingsModel,
+                    valueAttribute: "profile_image",
+                    editable: editable,
+                    imageMaxBytes: options['profile_image_max_bytes'],
+                    imageMinBytes: options['profile_image_min_bytes'],
+                    profileImageUploadUrl: options['profile_image_upload_url'],
+                    profileImageRemoveUrl: options['profile_image_remove_url']
```

Comment:
Can we just call these just `imageUploadUrl` and `imageRemoveUrl` since this `FieldView` may get used in other places too?